builtin/logical/transit/path_encrypt.go

```diff
@@ -63,46 +65,32 @@ func (b *backend) pathEncryptWrite(
 	// Decode the context if any
 	contextRaw := d.Get("context").(string)
 	var context []byte
+	var err error
 	if len(contextRaw) != 0 {
-		var err error
```

Comment:
I don't really like returning with stack variables except when it provides a huge cleanup benefit (for instance, when returning 4 or 5 items at once).